smartlearner/tasks/stopping_criteria.py

```diff
@@ -17,3 +19,31 @@ def __init__(self, nb_epochs_max):
     def post_epoch(self, status):
         if status.current_epoch >= self.nb_epochs_max:
             raise TrainingExit(status)
+
+
+class EarlyStopping(Task):
+    def __init__(self, cost, lookahead, callback=None, eps=0.):
```

Comment:
Add a `minimum_nb_epoch` to insure the training makes at least this much epochs before stopping.